docs/1.7/antora.yml

```diff
@@ -0,0 +1,42 @@
+name: os-manager
+title: OS Manager
+version: 1.7
+display_version: '1.7'
+start_page: en:index.adoc
+source-highlighter: highlight.js
```

Comment:
```suggestion
```

This isn't needed since highlight.js is [already the default](https://docs.antora.org/antora/latest/asciidoc/source/#source-highlighter). It's also something that should be set at the site-level if customization is needed.